src/Files.App/ViewModels/UserControls/Widgets/DrivesWidgetViewModel.cs

```diff
@@ -148,25 +148,29 @@ public override List<ContextMenuFlyoutItemViewModel> GetItemMenuItems(WidgetCard
 				{
 					Text = "Properties".GetLocalizedResource(),
 					OpacityIcon = new OpacityIconModel() { OpacityIconStyle = "ColorIconProperties" },
+					ShowItem = true,
 					Command = OpenPropertiesCommand,
 					CommandParameter = item
 				},
 				new()
 				{
 					Text = "TurnOnBitLocker".GetLocalizedResource(),
 					Tag = "TurnOnBitLockerPlaceholder",
-					IsEnabled = false
+					IsEnabled = false,
+					ShowItem = true,
 				},
 				new()
 				{
 					Text = "ManageBitLocker".GetLocalizedResource(),
 					Tag = "ManageBitLockerPlaceholder",
-					IsEnabled = false
+					IsEnabled = false,
+					ShowItem = true,
 				},
 				new()
```

Comment:
```suggestion
				new(ContextFlyoutItemType.Separator)
```
Would this work?

Reply:
I noticed you did something similar in `src/Files.App/ViewModels/UserControls/SidebarViewModel.cs`.